components/form/FormIcons.js

```diff
@@ -1,25 +1,47 @@
 import React from 'react';
 import classNames from 'classnames';
-import { HelpIcon, HintIcon, SuccessIcon } from '@/public/assets/dashboard/navBarIcon';
+import { HelpIcon, ToolTipIcon, SuccessIcon } from '@/public/assets/dashboard/navBarIcon';
+import ToolTip from '@/components/form/Tooltip';
 
-const FormIcons = ({ feedBack, hintIcon, isValid, hasErrors, isTextArea, isSelect }) => {
+const FormIcons = ({
+  feedBack,
+  floatLabel,
+  toolTip,
+  isValid,
+  hasErrors,
+  toolTipTitle,
+  toolTipContent,
+  toolTipColor,
+  toolTipPlacement,
+  showTooltipArrow,
+  isTextArea,
```

Comment:
As I have always said, defining props for each element is not sustainable and would be difficult to manage in the future. 

For instance, if we are adding Tooltip to the Datepicker component, we will need to add the props here, so this is not feasible.

Let's remove this file and use the tooltip component directly. We can use the positioning of each element to determine how it should look like. 

I will explain better during our sync.

Reply:
Resolve now